### ISSUE TITLE
Suspended chords

### ISSUE THREAD

**Issue by ldelia** [martijnversluis/ChordSheetJS#1166]
Hi, first of all, thanks again for this great library!

It looks like the ChordsOverWordsParser parser doesn't recognize suspended chords correctly. In the example above, instead of seeing Dsus4, the parser parses it as Dsus. Something similar happens when using suspended chords with a bass notation, e.g.,  Dsus4/G# will be parsed as Dsus/G#.

```
const chordSheet = `
       Am         C/G        Dsus4          C
Let it be, let it be, let it be, let it be
C                G              F  C/E Dm C
Whisper words of wisdom, let it be`.substring(1);

const parser = new ChordSheetJS.ChordsOverWordsParser();
const song = parser.parse(chordSheet);
```

**Comment by martijnversluis** [comment]
Hi @ldelia. Thanks for reaching out.

All formatters _should_ accept the `normalizeChords: false` option in the constructor, thus disabling normalization. However, that was not properly implemented in all formatters. I released `9.0.3` to fix that issue.

To format your parsed song without chord normalization:

```js
const formatter = new ChordsOverWordsFormatter({ normalizeChords: false }); # Or a different formatter
const formatted = formatter.format(song);
```

Please let me know if this fixes your issue!

**Comment by ldelia** [comment]
Hi @martijnversluis, thanks for your quick response. Now, I can see that I opened this ticket wrongly. I should have detailed that I was using the HtmlTableFormatter formatted, and thanks to your message, I can see that adding the { normalizeChords: false } arg in the constructor solves my issue. Everything works like a charm! Thanks!

![image](https://github.com/martijnversluis/ChordSheetJS/assets/1646735/6af166ba-dcf3-4758-9ffa-2ac1fad95b87)
